packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -516,8 +516,7 @@ class ScrollbarPainter extends ChangeNotifier implements CustomPainter {
     _lastMetrics = metrics;
     _lastAxisDirection = axisDirection;
 
-    bool needPaint(ScrollMetrics? metrics) => metrics != null && metrics.maxScrollExtent > metrics.minScrollExtent;
```

Comment:
Removing this public method would be a breaking change, can you revert that?

Reply:
Much like in [a previous PR](https://github.com/flutter/flutter/pull/154485#discussion_r1744810103) by @hgraceb, the narrow GitHub diffs make it kinda difficult to see the context of this change. Fortunately this is a local function defined inside the [**ScrollbarPainter.update**](https://main-api.flutter.dev/flutter/widgets/ScrollbarPainter/update.html) method, so I believe we're good 👍

Reply:
As @nate-thegrate said, thanks for the explanation.

Reply:
Oh it is indeed! Sorry I did not catch that! :)